docs/en/observability/synthetics-command-reference.asciidoc

```diff
@@ -55,3 +51,37 @@ Captures screenshots for every step in the journey.
 
 *`-h, --help`*::
 Shows help for the `npx @elastic/synthetics` command.
+
+[discrete]
+[[elastic-synthetics-init-command]]
+== `elastic-synthetics init`
+
+Scaffold a new project using Elastic Synthetics.
```

Comment:
Might be worth adding some additional information, something along the lines of:
> This will download example journeys and prepare the folder as a project [link to other docs] that can contain one or more monitors that will later be `push`ed to Kibana

Reply:
Done below.